network-area-diagram-viewer/src/main/resources/svg.js

```diff
@@ -0,0 +1,448 @@
+
+// TODO(Luma) consider three winding transformers
+// TODO(Luma) consider nodes with multiple buses
```

Comment:
Indeed... I realized in the meantime that we will need to redraw the annuli, we cannot rotate them: on the nodes linked to the moving node, if there is another edge which is not moving which is linked to an inner annulus, then the outer annulus might be completely different.

Reply:
added a TODO to the current code (maybe we could handle this in a separate PR)